packages/ui-react/src/components/SearchBar/SearchBar.tsx

```diff
@@ -18,25 +18,31 @@ export type Props = {
 
 const SearchBar: React.FC<Props> = ({ query, onQueryChange, onClearButtonClick, onClose, inputRef }: Props) => {
   const { t } = useTranslation('search');
+  const handleSubmit = (event: React.ChangeEvent<HTMLFormElement>) => {
+    event.preventDefault();
+    inputRef?.current?.blur();
```

Comment:
I think we should programmatically set the focus to the results. By using `#content` it applies to both the results page as "no results found".

```suggestion
    doucment.querySelector('#content')?.blur();
```

In DebatDirect we created an hidden "Skip to searchresults" after the searchbar. That is also a viable solution, though I find it less neat. I think we should keep the optimisation simple, because the OTT search functionality is really basic.

Reply:
The `#content` suggestion has my support! Should be a fairly quick fix:
- Add `id="search_results"` and `tabindex="-1"` to the div around the search results
- Use `document.querySelector('#search_results')?.focus();` here instead of the blur

Reply:
It is not possible to apply `id="search_results"` and `tabindex="-1"` to the `div` around the results, since that `div` is specified in the `LayoutGrid` (which doesn't know if it's a grid for the search or other instances). I can however introduce a new `div` around the search results to apply these properties, the focus is now correctly on the results after a search. 

**However**, whilst testing this, I found that the screenreader only says 'table' after focusing on the results. I want to give the users more info on what is selected, do you both agree an `aria-label={t('search_bar.search_label')}` will do the trick?

Reply:
You could add a prop to LayoutGrid with a default value, such as `id="layout_grid"`, then add it to the div, then use it for focus(). 

Maybe `"search_results": "Search results"` would be a better aria-label than "Search" I'd say. You need to add this as a prop to LayoutGrid as well then. If you're short in time, I think we can accept the extra div.

Reply:
1. Also a good option, I'll apply a more generic `id`
2. Ah excuse me, I meant the `search_bar.heading` translation, which is 'Search results', so I agree!

Reply:
Of course I meant to write:
```
document.querySelector('#content')?.focus();
```
(lazy me for not quality checking my previous reply)

>  'table' after focusing on the results.

I get different results when testing. Maybe it was because my previous reply was incomplete.
- Android: it works as expected (it focuses the container and it reads the title when searching for something that exists)
- IOS: it focusses the "skip to content" button. IDK why. But i consider this "good enough"

I think this is the most easier solution with low work effort. It has my preference.





The 'best' approach is using an `aria-labeledby` and point to the h2 of the searchResults or the h1 for the errorPage. But this could get quite tricky! Of course we can circumvent it with the suggestion you proposed.

Reply:
Thanks for your input Mike! It is weird you had other findings, because the `aria-label` or `aria-labelledby` approach cannot be ignored when implementing `document.querySelector('#content')?.focus();` because: on TalkBack only the words _"Table. Keyboard hidden."_ were spoken by the screenreader. I've applied `aria-labelledby` to account for this, see new commit.

Reply:
See my last comment: https://github.com/Videodock/ott-web-app/pull/138#issuecomment-1961283059

I think this conversation can be resolved.